Add energy blog (#24359)
home-assistant/home-assistant.github.io: source/_posts/2022-10-03-short-term-solutions-save-energy-and-money-europe.markdown

There are two ways of saving energy with heating. The first approach is to invest in more efficient technology to generate heat and improve the home insulation to stay warm. Things like getting a heat pump, improving insulation and getting thicker windows. Those things are not achievable before winter kicks in, so they are out of scope for this post.

The second way of saving energy is to use less of it. Governments have been advising to lower the overall temperature of your home, but putting the temperature too low can get a little too cold. Instead, you can be smarter about how you heat your home by installing smart thermostats, temperature sensors and thermostatic radiator valves (TRV).

<p class='img'>
<img src='/images/blog/2022-10-03-short-term-solutions-save-energy-and-money-europe/shelly-trv.png'>
Photo of a Shelly TRV
</p>

## Smart Thermostats

A thermostat works by measuring the temperature. If the current temperature is below the target temperature set by the user, it turns on the heating until the home is the right temperature.

If you’re renting your home, you can skip this section as upgrading a thermostat is generally not an option.

> The way a thermostat can help you save the most energy is by controlling the boiler, and preferably the boiler temperature. Heating the water is what drives the energy usage, so you wouldn’t want the boiler to heat the water to a higher temperature than that you would want your home to heat up to.

Reviewer: ```suggestion
The way a thermostat can help you save the most energy is by controlling the boiler, and preferably the boiler temperature. Heating the water is what drives the energy usage, so you wouldn’t want the boiler to heat the water to a higher temperature than is necessary to reach your target temperature.
```
